deployments/engine/helm/tiflow/templates/cases.yaml

```diff
@@ -28,6 +28,7 @@ spec:
             - "/tiflow-chaos-case"
             - "--duration=20m"
             - "--etcd-addr={{ include "tiflow.name" . }}-metastore-etcd:12479"
+            - "--business-meta-addr={{ include "tiflow.name" . }}-metastore-mysql:3306"
```

Comment:
Does this parameter duplicate with the configuration file?

Reply:
Yes, but parameters in config file will be overwritten in case of conflict.